webroot/results/admin/export_public.php

```diff
@@ -157,6 +154,10 @@ function exportPublic ( $sources ) {
         echo '.';  # shows both Apache and the user that the script is doing stuff and not hanging
       }
     }
+    //Check if any errors while fetching data
+    if (($error = mysql_error()) !== '') {
```

Comment:
Why stash the result of this function call in a `$error` variable if you don't go on to use it?